omoefe-dukuye/SendIT
# #162337948 panel backend feedback

#### What this PR does
*If merged, this pull request will*
- Improve error messages as suggested by panel
- Simplify response json
- Modify tests to conform to the new changes

#### How to test manually
- Clone the repo `git clone https://github.com/un-tethered/SendIT.git`
- Checkout the branch `ch-panel-backend-feedback-162337948`
- Install Postgresql
- Create a database named `SendIT`
- Run `npm install` to install application dependencies
- Inspect code base
- Use Postman to check endpoints for improvements
- Run `npm test` to run unit tests

#### Pivotal tracker story
[#162337948](https://www.pivotaltracker.com/story/show/162337948)
